nrf52-hal-common/src/lib.rs

```diff
@@ -31,6 +31,18 @@ pub mod prelude {
     pub use crate::uarte::UarteExt;
 }
 
+/// Length of Nordic EasyDMA differs for MCUs
+#[cfg(feature = "52832")]
+pub mod target_constants{
```

Comment:
```suggestion
pub mod target_constants {
```

Formatting suggestion